### ISSUE TITLE
Allow install profiles to define dependencies other than modules

### ISSUE THREAD

**Issue by BWPanda** [backdrop/backdrop-issues#3926]
Install profiles can define dependencies: modules that are required before the profile can be installed (see https://www.drupal.org/docs/7/creating-distributions/how-to-write-a-drupal-7-installation-profile). I'd like to suggest allow dependencies to cover other types or projects as well (e.g. themes and layouts).

**Comment by stpaultim** [comment]
Yes, I was just trying to set Tatsu as the default theme in my install profile and it appears that this is not working. 

Any idea how difficult this would be to fix? I might be willing to work on this, but I don't know where to start.

**Comment by stpaultim** [comment]
Is there another way to "install" a theme in a install profile, other than declaring it as a dependency?

**Comment by BWPanda** [comment]
Yeah, that's the solution for now. It just doesn't have the nice validation (preventing install if theme is missing) that dependencies come with.

**Comment by BWPanda** [comment]
FTR, https://github.com/backdrop/backdrop-issues/issues/3924 has been marked as 'blocked' by this issue.

**Comment by shabib-m** [comment]
I suggest adding a feature to select the theme via the .info file using themes[] = basis, instead of using PHP code.

**Comment by quicksketch** [comment]
@shabib-m That seems like a good idea, though potentially we'll need another key for `default_theme = basis` or something, since just enabling a theme does not make it the default.
